Readme update (#71)
ros-industrial/abb_libegm: README.md

[![license - bsd 3 clause](https://img.shields.io/:license-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
[![support level: vendor](https://img.shields.io/badge/support%20level-vendor-brightgreen.png)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)

## Important Notes

RobotWare versions less than `6.07.01` are now incompatible with *abb_libegm* (due to changes in the EGM communication protocol).

> Pull request https://github.com/ros-industrial/abb_libegm/pull/63 turned this package from a Catkin package, into a plain CMake package. ROS users can utilize any of the following build tools to build the source code:

Reviewer: ```suggestion
Pull request [ros-industrial/abb_libegm#63](https://github.com/ros-industrial/abb_libegm/pull/63) turned this package from a Catkin package into a plain CMake package. ROS users may use any of the following build tools to build the library:
```

Reviewer: This actually also contains some updates to the rest of the sentence. Somehow I'd 'fixed' the way the PR link is displayed twice.

Author: I saw them and they should already have been added.

[![license - bsd 3 clause](https://img.shields.io/:license-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
[![support level: vendor](https://img.shields.io/badge/support%20level-vendor-brightgreen.png)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)

## Important Notes

RobotWare versions less than `6.07.01` are now incompatible with *abb_libegm* (due to changes in the EGM communication protocol).

> Pull request [#63](https://github.com/ros-industrial/abb_libegm/pull/63) turned this package from a Catkin package, into a plain CMake package. ROS users can utilize any of the following build tools to build the source code:

Reviewer: ```suggestion
Pull request [abb_libegm#63](https://github.com/ros-industrial/abb_libegm/pull/63) turned this package from a Catkin package, into a plain CMake package. ROS users can utilize any of the following build tools to build the source code:
```

* ROS 1: `catkin_make_isolated` or [catkin_tools](https://catkin-tools.readthedocs.io/en/latest/index.html).

> * ROS 2: `colcon`.

Reviewer: ```suggestion
* ROS 2: [colcon](https://colcon.readthedocs.io/en/released/).
```

## Overview

> A C++ library for interfacing with ABB robot controllers supporting *Externally Guided Motion* (EGM). See the *Application manual - Externally Guided Motion* (document ID: `3HAC073319-001`, revision: `B`) for a detailed description of what EGM is and how to use it.

Reviewer: Is this document publicly available? I cannot find anything with that document nr.

Author: Hm... I can find it in the `ABB Library`, but it is tagged as `PREMIUM` so I don't think it is publicly available. I usually access the `ABB Library` via RobotStudio:

![Capture](https://user-images.githubusercontent.com/16589371/73663964-e0e36f00-469e-11ea-9ebd-40e6855d079c.PNG)

Reviewer: As long as users of this library would have access to it.

Just wanted to make sure.

Author: I can't imagine that users wouldn’t have access to the documentation if they have purchased the EGM license.

## Overview

> A C++ library for interfacing with ABB robot controllers supporting *Externally Guided Motion* (EGM). See the *Application manual - Externally Guided Motion* for a detailed description of what EGM is and how to use it.

Reviewer: Does this have a document number? If so, I suggest we add it (including the version/revision).

Author: Yes, it has, and I will add it.
